packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.jsx

```diff
@@ -56,11 +55,14 @@ const UploaderSideNote = () => (
 const ProofOfAddressForm = ({
     account_settings,
     addNotificationByKey,
+    index = null,
```

Comment:
index type is declared as number. can you check this?

Reply:
yeapp it's number will change it to index = 0